application/F3DOptionsParser.cxx

```diff
@@ -772,3 +778,15 @@ void F3DOptionsParser::LoadPlugins(const F3DAppOptions& appOptions) const
 {
   return this->ConfigOptions->LoadPlugins(appOptions);
 }
+
+//----------------------------------------------------------------------------
+void F3DOptionsParser::WarnForDeprecatedOptions(const F3DAppOptions&, const f3d::options& options)
```

Comment:
in F3DOptionsParser::GetOptions ? This will be very spammy

Reply:
Maybe in `F3DStarter::Start`, just after the call to `GetOptions`?

Reply:
then we need to check the options, why not in a dedicated method ? Indeed it could be namespaced but I like that its in F3DOptionParser instead of being in F3DStarter

Reply:
> Maybe in F3DStarter::Start, just after the call to GetOptions?

We need to check just after loading the file, as we need to check for deprecated usage in the config files

I stand by the idea of putting that in a dedicated method.

Reply:
OK, then what's the point of declaring a new function with an argument you are not using?

Reply:
not absolutely needed for now, true, but will be needed once we deprecate options from F3DAppOptions. Ok to remove if you prefer.

Reply:
yes I think it's better to remove it add we'll add it later if needed.